test/lib/jdk/test/lib/jfr/GCHelper.java

```diff
@@ -92,20 +93,22 @@ public static boolean isGcEvent(RecordedEvent event) {
         return false;
     }
 
-//    public static String getEventDesc(RecordedEvent event) {
-//      final String path = event.getEventType().getName();
-//        if (!isGcEvent(event)) {
-//            return path;
-//        }
-//        if (event_garbage_collection.equals(path)) {
-//            String name = Events.assertField(event, "name").getValue();
-//            String cause = Events.assertField(event, "cause").getValue();
-//            return String.format("path=%s, gcId=%d, endTime=%d, name=%s, cause=%s, startTime=%d",
-//                    path, getGcId(event), event.getEndTime(), name, cause, event.getStartTime());
-//        } else {
-//            return String.format("path=%s, gcId=%d, endTime=%d", path, getGcId(event), event.getEndTime());
-//        }
-//    }
+    // public static String getEventDesc(RecordedEvent event) {
+    // final String path = event.getEventType().getName();
+    // if (!isGcEvent(event)) {
+    // return path;
+    // }
+    // if (event_garbage_collection.equals(path)) {
+    // String name = Events.assertField(event, "name").getValue();
+    // String cause = Events.assertField(event, "cause").getValue();
+    // return String.format("path=%s, gcId=%d, endTime=%d, name=%s, cause=%s,
+    // startTime=%d",
+    // path, getGcId(event), event.getEndTime(), name, cause, event.getStartTime());
+    // } else {
+    // return String.format("path=%s, gcId=%d, endTime=%d", path, getGcId(event),
+    // event.getEndTime());
+    // }
+    // }
```

Comment:
This looks whitespace only changes that are semantically no-ops. Back this out as it clutters the diffs.